src/hashp/core.clj

```diff
@@ -34,11 +36,19 @@
 (defn p* [form]
   (let [orig-form (walk/postwalk hide-p-form form)]
     `(let [~result-sym ~form]
-       (locking lock
-         (println
-          (str prefix
-               (color/sgr (trace-str (current-stacktrace)) :green) " "
-               (when-not (= ~result-sym '~orig-form)
-                 (str (puget/pprint-str '~orig-form print-opts) " => "))
-               (puget/pprint-str ~result-sym print-opts)))
-         ~result-sym))))
+       (macrovich/case
+           :clj (locking lock
+                  (println
+                   (str prefix
+                        (color/sgr (trace-str (current-stacktrace)) :green) " "
+                        (when-not (= ~result-sym '~orig-form)
+                          (str (puget/pprint-str '~orig-form print-opts) " => "))
+                        (puget/pprint-str ~result-sym print-opts)))
+                  ~result-sym)
+           :cljs (do
+                   (println
+                    (str prefix " "
+                         (when-not (= ~result-sym '~orig-form)
+                           (str (zprint/zprint-str '~orig-form print-opts) " => "))
+                         (zprint/zprint-str ~result-sym print-opts)))
+                   ~result-sym)))))
```

Comment:
The indentation here is four spaces instead of 2.

Reply:
done!